### DISCUSSION TITLE
Missing `fetching` property on URQL queries

### DISCUSSION THREAD

**Discussion by carlesba** [pmndrs/jotai#1226] [Q&A] [answered]
I can't find `fetching` in the value provided by `atomWithQuery`. 

The property is documented [here](https://formidable.com/open-source/urql/docs/api/urql/)

Is there any workaround?

**Comment by dai-shi** [comment, marked as answer]
jotai atoms are suspense first. So, atomWithQuery doesn't provide such a property.
You can use `loadable` util to get loading status for any async atoms.
